.github/workflows/tests.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.4, 7.3, 7.2, 8.0]
```

Comment:
You may preserve all these if you want to remain backwards compatible. Although I would recommend dropping PHP 7.2. 😉

Reply:
Both PHP 7.2 and 7.3 are no longer maintained so we can probably drop the support for them if we're doing a new major version but if we want to go with a minor version for compatibility with L9 I would keep all of these PHP versions.